README.md

```diff
@@ -247,7 +287,7 @@ The following logging functions can be used:
 
 ### Adding SSH support to a repository
 
-See [this](ci/README.md)
+- For Travis CI, see [this](docs/CI_Travis_Setup.md)
```

Comment:
This also contains one header and line for Gitlab.

Reply:
I do not understand this comment

Reply:
This was for GitLab when we had moved hero-setup there to place Toyota specific things in it.